test/karma/package.json

```diff
@@ -5,18 +5,18 @@
   "types": "test-dist/types/components.d.ts",
   "collection": "test-dist/collection/collection-manifest.json",
   "scripts": {
-    "build.app": "node ../../bin/stencil build --debug --es5",
+    "build.app": "npm run compile.test-app && node ../../bin/stencil build --debug --es5",
+    "compile.test-app": "node ../../node_modules/typescript/lib/tsc -p tsconfig.json",
```

Comment:
This was originally under a script named `tsc` - at first read I thought it was invoking the TS compiler directly, so I renamed it for clarity. This appears to only be directly used for the test-app, hence the name.  I did consider moving the `tsconfig.json` itself down to the `test-app/`, but decided against it as it's referenced in other places and felt out of scope of this user story